lenovo/thinkpad/t480s/default.nix

```diff
@@ -9,4 +14,10 @@
   ];
 
   services.throttled.enable = lib.mkDefault true;
+
+  # There is currently a kernel regression with the psmouse driver. As such, the kernel version is held back to 6.10.
+  # See https://bugzilla.kernel.org/show_bug.cgi?id=219352
+  boot.kernelPackages = lib.mkIf (lib.versionAtLeast config.boot.kernelPackages "6.11") (
+    lib.warn "Linux kernel held back to 6.10 due to regression: https://bugzilla.kernel.org/show_bug.cgi?id=219352" pkgs.linuxPackages_6_10
```

Comment:
Should we not just warn in this case? The 6.10 kernel might get removed at this point.

Reply:
The default kernel right now is 6.6 iirc, so I doubt that's gonna happen.

Reply:
The kernel package referenced here is 6.10. It's not an lts kernel. But the actual issue of this code is that it's an infinite recursion. You are checking `config.boot.kernelPackage` while writing to the same option.

Reply:
Yea, this hasn't been tested. TBH I completely forgot about this PR; I somehow forgot to put it on my tracking board after opening it.